### PR TITLE
Remove the last dependency for mootools in CMS Installation

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### We use mootools only for a spinner???

We can do it better and also using the Joomla! spinner logo.
#### Testing

d/l this https://github.com/dgt41/joomla-cms/archive/noMootoolsInstall.zip
unzip and install as usual

### CODE REVIEW

**Comment by Bakual** [comment]
Just wondering if we can't use https://github.com/joomla/joomla-cms/blob/staging/media/jui/images/ajax-loader.gif instead of adding the same spinner also into the installation folder :smile:

File: installation/template/js/installation.js [hunk 1]
```diff
@@ -18,6 +18,21 @@ var Installation = function(_container, _base) {
         });
     }
 
+    var spinnerShow = function() {
+        jQuery('#loading')
```
Comment (piotr-cz) [inline installation/template/js/installation.js]: Why redeclaring all these styles, if these are already present in `template.css`?

**Comment by Bakual** [comment]
Works fine, however the spinner is a bit far on top I think. The old one was more in the center of the page and thus more visible.

**Comment by photodude** [comment]
:+1:

**Comment by dgt41** [comment]
@Bakual spinner back in the middle of the page

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on c7b10b0935718eba0ae360ca98df1eb7a9b01495<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8514">issues.joomla.org/joomla-cms/8514</a>.</sub>

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on c7b10b0935718eba0ae360ca98df1eb7a9b01495<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8514">issues.joomla.org/joomla-cms/8514</a>.</sub>

**Comment by dgt41** [comment]
@joomla-bot are you there?

**Pull request merged**